kuma/static/styles/components/payments/page.scss

```diff
@@ -64,6 +72,13 @@
     .faqs {
         flex: 60%;
 
+        a {
+            border-bottom: 1px solid $payment-form-button-color;
```

Comment:
This surprises me. What's the reason for a border instead of a text-decoration? Is this related to the comment "same as legal copy links"?

Reply:
Yup, I found it strange as well. I did not want to mess too much with the existing code other than fixing what was required. Not sure why the Potato devs decided to go this route. Something I will look into as I clean up the SaSS